scene/gui/text_edit.cpp

```diff
@@ -2902,6 +2923,30 @@ int TextEdit::cursor_get_line() const {
 	return cursor.line;
 }
 
+bool TextEdit::cursor_get_blink_enabled() const {
+	return caret_blink_enabled;
+}
+
+void TextEdit::cursor_set_blink_enabled(const bool p_enabled) {
```

Comment:
Should those functions be exposed to GDScript too to be available in games using TextEdit, or tool script extending editor functionalities maybe?